control_plane/src/storage.rs

```diff
@@ -321,11 +322,20 @@ impl PageServerNode {
             .json()?)
     }
 
-    pub fn tenant_create(&self, tenantid: ZTenantId) -> Result<()> {
+    pub fn tenant_create(&self, tenant_id: ZTenantId, settings: HashMap<&str, &str>) -> Result<()> {
         Ok(self
             .http_request(Method::POST, format!("{}/{}", self.http_base_url, "tenant"))
             .json(&TenantCreateRequest {
-                tenant_id: tenantid,
+                tenant_id,
+                checkpoint_distance: settings
+                    .get("checkpoint_distance")
+                    .map(|x| x.parse::<u64>().unwrap()),
+                checkpoint_period: settings.get("checkpoint_period").map(|x| x.to_string()),
+                gc_horizon: settings
+                    .get("gc_horizon")
+                    .map(|x| x.parse::<u64>().unwrap()),
+                gc_period: settings.get("gc_period").map(|x| x.to_string()),
```

Comment:
see [Option::copied](https://doc.rust-lang.org/nightly/std/option/enum.Option.html#method.cloned)

Reply:
Oh, it's shame that `cloned` doesn't cover this case. You can use `.map(String::from)` but this has almost the same length